### PR TITLE
Remove the old table @mixin with the new one

### PR DESCRIPTION
**Pull request by RazvanDH**
- removed the old table mixin with a new one that has lower specificity. 
- renamed the old one to @mixin tableOld - we might still use it for smaller tasks

### CODE REVIEW

File: dist/sass/_mixins/structure.scss [hunk 1]
```diff
@@ -77,7 +77,7 @@
                         | vertical-align: top
 \* ---------------------------------------------------------------------- */
 
-@mixin table($verticalAlign:top, $reset:false, $hide:false, $cols: false, $row:false, $padding: $defaultColsPadding) {
+@mixin tableOld($verticalAlign:top, $reset:false, $hide:false, $cols: false, $row:false, $padding: $defaultColsPadding) {
```
Comment (igoratron) [inline dist/sass/_mixins/structure.scss]: can we give it a name that reflects it's purpose? If it's deprecated then lets just remove it.

File: dist/sass/_mixins/structure.scss [hunk 2]
```diff
@@ -256,7 +256,7 @@ $colRatio: 1;
 }
 
 
-@mixin tableV2($verticalAlign:top, $reset: false, $hide: false, $cols: false, $row:false, $padding: $defaultColsPadding) {
+@mixin table($verticalAlign:top, $reset: false, $hide: false, $cols: false, $row:false, $padding: $defaultColsPadding) {
```
Comment (igoratron) [inline dist/sass/_mixins/structure.scss]: :+1: for not naming it `--t1` :stuck_out_tongue_winking_eye:

File: dist/sass/_mixins/structure.scss (hunk 1 above)
Comment (RazvanDH) [inline dist/sass/_mixins/structure.scss]: Hm, you're right. I've removed it.

File: dist/sass/_mixins/structure.scss (hunk 2 above)
Comment (RazvanDH) [inline dist/sass/_mixins/structure.scss]: :laughing:

**Comment by igoratron** [comment]
Just bump the package version and I am happy :grinning:

**Comment by RazvanDH** [comment]
Package version updated! After merging with master we should add a new tag and update static-content.

**Comment by igoratron** [comment]
Thanks @RazvanDH

**Pull request merged**